model/proxy_service_notify.go

```diff
@@ -0,0 +1,18 @@
+package model
+
+// PerformanceTestResultID is the ID to uniquely identify a new performance result available at test level.
+type PerformanceTestResultID struct {
+	Version   string           `json:"version"`
+	TaskID    string           `json:"task_id"`
+	Project   string           `json:"project"`
+	Variant   string           `json:"variant"`
+	Task      string           `json:"task"`
+	Test      string           `json:"test"`
+	Arguments map[string]int32 `json:"args"`
+}
+
+type PerformanceAnalysisProxyServiceOptions struct {
```

Comment:
Documentation.

Reply:
Also, I think this belongs in `perf/proxy_service_notifier.go` since it is the options struct for the client creator func there.